docs/tutorials/tutorial-sql-query-view.md

```diff
@@ -0,0 +1,168 @@
+---
+id: tutorial-sql-query-view
+title: "Tutorial: Use Query view"
+sidebar_label: "Use Query view"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+
+This tutorial demonstrates some useful features of the Query view in Apache Druid.
+
+You can use the Query view to ingest and query data. You can also use it to test and tune queries before you use them in API requests&mdash;for example, to perform [SQL-based ingestion](../multi-stage-query/api.md).
```

Comment:
I think the tone here should be that querying is the main thing you do in the Query view and you can also use it to ingest data (via SQL based ingestion). I know the tutorial later starts by ingesting data first to be self contained (and that is fine). I am talking about the introductory text.

I would also mention that you can run JSON based queries* from the Query view.

* these are sometimes called "native queries" but I am trying to use the term "native" less as I do not think it is very descriptive to an outsider

Reply:
@vogievetsky , It is a little late in the game to change terms for "native" queries at this point. What if we just add the qualifier "JSON-based" to "native queries" and link out to the doc for clarification: https://druid.apache.org/docs/latest/querying/querying.html#available-queries

Reply:
Updated the intro